frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,7 +172,13 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
-  const resultDataVolumeName = prefixedID(vmName, name);
+  // We can generate a random name every time, because this modal should not operate on disks with live datavolumes
+  const resultDataVolumeName = resolveDataVolumeName({
+    diskName: name,
+    vmLikeEntityName: vmName,
+    isTemplate: isCreateTemplate,
```

Comment:
the same problem as mentioned in https://github.com/openshift/console/pull/5618#discussion_r436117134 still persists.
We are not passing `isCreateTemplate` variable in the details flow.

We should pass this variable, but the fix is not so simple. Due to the use of `isPlainDataVolume` - which should take effect only in the wizard.

Let's try splitting `isCreateTemplate` into two variables: `isTemplate` and `isInWizard` and resolve datavolumeName and plainDataVolume warning accordingly.